docs/modules/ROOT/pages/tutorial-onpremise.adoc

```diff
@@ -1,32 +1,39 @@
-= On-premise example
-
-[abstract]
-CMOS is deployed to monitor an on-premise cluster with a simple Nginx web server exposed on port 8080 to provide a landing page.
-
-include::partial$tutorial.adoc[]
+= CMOS On Premise Deployment
 
 .Microlith overview
 image::microlith-runtime.png[]
 
-== Usage
+CMOS is deployed to monitor an on-premise cluster with a simple Nginx web server exposed on port 8080 to provide a landing page.
 
-To run refer to the xref:quickstart.adoc[quick start guide] for full details:
+== Deployment
 
+At this moment we only support deployment using docker. You can get started by deploying the docker container:
```

Comment:
== Accessing the deployed services
There are two ways to access the services deployed in the container
=== Browser
A landing page provided by the cmos stack through nginx has links to all the deployed services. You can navigate to <host_ip>:8080 to access the landing page.

=== Direct port access
You can directly access the service via exposed ports either through browser or through any other consuming service. e.g. to access grafana which is exposed on port 3000 you can navigate to <ip>:3000 in your browser.

In most of the cases you might endup setting port forwarding or port exposuire within your network, exposing ports of the runing docker container has no special steps and you can follow the general guidelines used in you r organisation.